typings/styled-components.d.ts

```diff
@@ -12,7 +12,7 @@ export type StyledProps<P> = ThemedStyledProps<P, any>;
 
 export type ThemedOuterStyledProps<P, T> = P & {
   theme?: T;
-  innerRef?: (instance: any) => void;
+  innerRef?: ((instance: object) => void) | RefObject<HTMLElement | SVGElement>
```

Comment:
I think we still need it be 
```ts
innerRef?: ((instance: object) => void) | RefObject<object>
```
as it can be a ref to react component instance

Reply:
You are correct. Still, `object` is very unspecific. Same as in `(instance: object) => void`.

There's got to be something from `@types/react` that we can use that is more specific, right? Something along the lines of `RefObject<HTMLElement | SVGElement | Component<{}, {}, any>>`?

Reply:
yep, `Component` should work 👍

Reply:
Done! 😃 👍